### ISSUE TITLE
Model::validate getRelated 

### ISSUE THREAD

**Issue by rudiservo** [phalcon/cphalcon#12089]
Issue with validate if the Model has a property named 'related'.

it tries to run the getValue() method looking for a method of the propertie getRelated() that already exists in the Model Class.

Solution may pass by putting a changing the Model class function to _getRelated();

(thanks to @Jurigag that figured out the issue).

```
PHP Fatal error:  Uncaught exception 'BadMethodCallException' with message 'Wrong number of parameters' in /var/www/v2/app/models/Files.php:124
Stack trace:
#0 [internal function]: Phalcon\Mvc\Model->getRelated()
#1 [internal function]: Phalcon\Validation->getValue('RELATED')
#2 [internal function]: Phalcon\Validation\Validator\Uniqueness->isUniqueness(Object(Phalcon\Validation), Array)
#3 [internal function]: Phalcon\Validation\Validator\Uniqueness->validate(Object(Phalcon\Validation), Array)
#4 [internal function]: Phalcon\Validation->validate(NULL, Object(Files))
#5 /var/www/v2/app/models/Files.php(124): Phalcon\Mvc\Model->validate(Object(Phalcon\Validation))
#6 [internal function]: Files->validation()
#7 [internal function]: Phalcon\Mvc\Model->fireEventCancel('validation')
#8 [internal function]: Phalcon\Mvc\Model->_preSave(Object(Phalcon\Mvc\Model\MetaData\Memory), false, 'id')
#9 /var/www/v2/app/tasks/FileTask.php(346): Phalcon\Mvc\Model->save()
#10 /var/www/v2/app/tasks/File in /var/www/apqe/v2/app/models/Files.php on line 124

```

``` php
class Files extends Model
{
    public $related;

    public function validation()
    {
        $validator = new Validation();
        $validator->add('related', new Uniqueness());
        return $this->validate($validator);
    }
}
```
### Details
- Phalcon version: 3

**Comment by andresgutierrez** [comment]
You can rename that column using: https://docs.phalconphp.com/en/latest/reference/models.html#independent-column-mapping

**Comment by rudiservo** [comment]
@andresgutierrez Thank you

**Comment by Jurigag** [comment]
How you want to fix this in 3.0.1 ? Change validate method somehow to not use getters ? Or to check somehow if it's method from `Phalcon\Mvc\Model` ? @sergeyklay
